_data/simple-icons.json

```diff
@@ -4525,6 +4525,12 @@
             "source": "https://www.juniper.net/us/en/company/press-center/images/image-library/logos/",
             "guidelines": "https://www.juniper.net/us/en/company/press-center/images/image-library/logos/"
         },
+        {
+            "title": "JUnit5",
+            "hex": "25A162",
+            "source": "https://github.com/junit-team/junit5/tree/86465f4f491219ad0c0cf9c64eddca7b0edeb86f/assets/img",
+            "guidelines": "https://github.com/junit-team/junit5/tree/86465f4f491219ad0c0cf9c64eddca7b0edeb86f/assets/img"
```

Comment:
Given there are no explicit guidelines at the address, I suggest we remove them.
```suggestion

```